changelog.md

```diff
@@ -39,6 +39,60 @@ would assign these ports as host ports if available in the
 processed offer. That misled people into thinking that these ports corresponded to host ports. The new code always
 randomizes host ports assignment without `"requirePorts"` or explicit `"hostPort"` configuration.
 
+##### Additional task statistics in the `/v2/apps` and `/v2/apps/{app id}` endpoints (Optional)
+
+If you pass `embed=apps.taskStats`/`embed=app.taskStats` as a query parameter, you get additional taskStats
+embedded in you app JSON.
+ 
+Task statistics are provided for the following groups of tasks. If no tasks for the group exist, no statistics are
+offered:
+
+* `"withLatestConfig"` contains statistics about all tasks that run with the same config as the latest app version.
+* `"startedAfterLastScaling"` contains statistics about all tasks that were started after the last scaling or
+  restart operation.
+* `"withOutdatedConfig"` contains statistics about all tasks that were started before the last config change which
+  was not simply a restart or scaling operation.
+* `"totalSummary"` contains statistics about all tasks.
+
+Example JSON:
+
+```javascript
+{
+  // ...
+  "taskStats": {
+    {
+      "startedAfterLastScaling" : {
+        "stats" : {
+          "counts" : { // equivalent to tasksStaged, tasksRunning, tasksHealthy, tasksUnhealthy
+            "staged" : 1,
+            "running" : 100,
+            "healthy" : 90,
+            "unhealthy" : 4
+          },
+          "lifeTime" : { // measured from the "stagedAt" timestamp of Mesos
```

Comment:
Why `stagedAt` as opposed to the timestamp of a `TASK_RUNNING` event? I've seen tasks being `STAGED` for a long time (due to the slave dropping out and not re-registering). These staging tasks would raise the overall lifetime although they probably won't ever be running before they'll eventually be killed (after the slave eventually reconnects …)
Maybe I'll get the reason when I see the implementation, so don't mind if it's obvious ;)